### ISSUE TITLE
Extra transient at end of imported granular samples

### ISSUE THREAD

**Issue by radamJ** [raspy135/Qun-mk2#14]
Every time I import a sample into granular, it has an extra transient at the end. All the samples I use are mono 48kHz/16bit WAV.

It doesn't seem like I could take screenshots of the granular screen but I attached a couple photos. One shows three separate samples I imported (a kick, snare and hi hat). You can see a small transient at the end of each. The other image is just zoomed in on the first sample to make it more obvious what I am referring to.

![Qun](https://github.com/raspy135/Qun-mk2/assets/4553740/8da7f0b8-15fd-4d8a-99cd-c0508315c44d)
![Qun Zoomed](https://github.com/raspy135/Qun-mk2/assets/4553740/9abafb5c-976d-4ecc-b11c-f7acbb083925)

**Comment by raspy135** [comment]
I loaded multiple samples for the test but I couldn't reproduce the issue.
Did you load short sample and you hear the pop noise or something at the end?

**Comment by radamJ** [comment]
Yes, they were short samples and I hear a pop at the end. It is happening consistently for me with all of my samples. Is there an email address you’d like me to send some of the samples to so you for testing?

**Comment by raspy135** [comment]
Can you send the wav files DM on Discord if you already joined Discord?

**Comment by radamJ** [comment]
Sure, I can do that tomorrow
